server/test/services/applicant/ReadOnlyApplicantProgramServiceImplTest.java

```diff
@@ -73,6 +73,47 @@ public void getProgramTitle_returnsProgramTitleForPreferredLocale() {
     assertThat(subject.getProgramTitle()).isEqualTo("Mein Programm");
   }
 
+  @Test
+  public void getStoredFileKeys_includesAnsweredFileQuestions() {
+    QuestionDefinition fileQuestionDefinition =
```

Comment:
Add one for includes legacy files

Reply:
I'm not sure what that means in the context of this class/test. This class doesn't have logic that differentiates between the two, nor does it deal directly with the `StoredFile` class.